client/reader/following-manage/index.jsx

```diff
@@ -198,7 +204,7 @@ class FollowingManage extends Component {
 							/>
 						</div> }
 				</div>
-				{ ! sitesQuery && <RecommendedSites sites={ take( recommendedSites, 2 ) } /> }
+				{ hasFollows && ! sitesQuery && <RecommendedSites sites={ take( recommendedSites, 2 ) } /> }
```

Comment:
i think it's a combo of their likes / follows / magical other stuff. @gibrown @aaronyan any thoughts on this? should we show recs to a new user?